identity/roles.go

```diff
@@ -189,3 +190,81 @@ func (c *RolesClient) Delete(ctx context.Context, input *DeleteRoleInput) error
 
 	return nil
 }
+
+// RoleTags
+type RoleTags struct {
+	// Path to the resource.
+	Name string `json:"name"`
+	// The list of role tags assigned to this resource.
+	RoleTags []string `json:"role-tag"`
+}
+
+// GetRoleTagsInput
```

Comment:
comment on exported type GetRoleTagsInput should be of the form "GetRoleTagsInput ..." (with optional leading article)